pvlib/atmosphere.py

```diff
@@ -264,28 +263,31 @@ def gueymard94_pw(temp_air, relative_humidity):
 
     .. math::
 
-           H_v = 0.4976 + 1.5265*T/273.15 + \exp(13.6897*T/273.15 - 14.9188*(T/273.15)^3)
+           H_v = 0.4976 + 1.5265 \frac{T}{273.15} +
+               \exp \left(13.6897 \frac{T}{273.15} -
```

Comment:
Prefer leading operator v trailing, also useful for `\begin{aligned}` but personal style not a blocker